src/v3/src/util/formUtils.ts

```diff
@@ -34,6 +36,10 @@ import {
 import { idpIconMap } from './idpIconMap';
 import { loc } from './locUtil';
 
+export type PhoneVerificationStep = typeof IDX_STEP.CHALLENGE_AUTHENTICATOR
+| typeof IDX_STEP.AUTHENTICATOR_VERIFICATION_DATA;
+export type PhoneVerificationMethodType = 'sms' | 'voice';
```

Comment:
Any way we can take advantage of this type in my other comment about the existence of a union type?

Reply:
Oh, I can use it in the assertion part of that function, but for the parameter, I don't think I can use it in the type there unless I cast the parameter value before passing (which defeats the purpose of the guard). Unless you were thinking of something else?